PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionExDialog.java

```diff
@@ -40,6 +41,8 @@
 import java.util.Optional;
 
 public class SparkSubmissionExDialog extends JDialog {
+    private static final Key<SparkSubmitModel> SUBMISSION_DATA_KEY = Key.create("azure.hdinsight.spark.job.submission");
+
```

Comment:
Move this to "SparkSubmissionContentPanelConfigurable", so when users change cluster in "project right click", it will be reflected in run configuration as well?